applications/nrf_desktop/README.rst

```diff
@@ -1857,6 +1857,46 @@ These are valid for events that have many listeners or sources, and are gathered
    doc/hfclk_lock.rst
    doc/event_rel_modules.rst
 
+API documentation
+*****************************
+
+Following are the API elements used by the application configuration.
+
+HID reports
+===========
+
+| Header file: :file:`applications/nrf_desktop/configuration/common/hid_report_desc.h`
+| Source file: :file:`applications/nrf_desktop/configuration/common/hid_report_desc.h`
+
+.. doxygengroup:: nrf_desktop_hid_reports
+   :project: nrf
+   :members:
+
+LED states
+==========
+
+| Header file: :file:`applications/nrf_desktop/configuration/common/led_state.h`
+| Source file: :file:`applications/nrf_desktop/src/modules/led_state.c`
+
+.. doxygengroup:: nrf_desktop_led_state
+   :project: nrf
+   :members:
+
+Application events API
+**********************
```

Comment:
We could remove this heading as it is aligned to 'API documentation' heading which is bit misleading.

Reply:
done